tests/k8s/client.yml

```diff
@@ -0,0 +1,22 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  generateName: client-
+spec:
+  selector:
+    matchLabels:
+       run: client
+  replicas: 5
+  template:
+    metadata:
+      labels:
+        run : client
+    spec:
+      containers:
+      - name: client
+        image: spacemeshos/go-spacemesh:ms1
```

Comment:
the image should be parametrized, we can't expect each dev to replace yaml file params each tests he runs. same goes for the `args` below.

Reply:
This file is used as a template for the deployment. This parameters have the most general values.
Some of them are override in the test execution with values from a config file (config.yaml)